recipes/default.rb

```diff
@@ -28,6 +28,16 @@
   end
 end
 
+# Create the mongodb user if not exist
+user node['mongodb3']['user'] do
```

Comment:
Yes, you are right. But I've got confused because the cookbook doesn't work when attribute default['mongodb3']['user'] != 'mongodb'. I just left with default value and everything works.